swap_meet/vendor.py

```diff
@@ -1,2 +1,72 @@
+from swap_meet.item import Item
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory
+        if self.inventory is None:
+            self.inventory = []
+    def add(self, item):
+        self.item = item
+        self.inventory.append(item)
+        return self.item
```

Comment:
Thank you! this clears up a lot of confusion. I wasn't sure when we add self. in front of a parameter and when we don't. My tutor went over this with me today so that cleared up some of the confusion